src/group_theory/submonoid/membership.lean

```diff
@@ -294,6 +294,23 @@ theorem log_pow_int_eq_self {x : ℤ} (h : 1 < x.nat_abs) (m : ℕ) : log (pow x
   (powers m).map f = powers (f m) :=
 by simp only [powers_eq_closure, f.map_mclosure, set.image_singleton]
 
+/-- If all the elements of a set `s` commute, then `closure s` forms a commutative monoid. -/
```

Comment:
```suggestion
/-- If all the elements of a set `s` commute, then `closure s` is a commutative monoid. -/
```
and similarly elsewhere. (My English is terrible, so don't hesitate to ignore my comment if "forms" is better).